res/skins/Tango (64 Samplers)/skin.xml

```diff
@@ -26,8 +26,9 @@
   <manifest>
     <title>Tango</title>
     <author>ronso0@mixxx.org</author>
-    <version>2.3.0</version>
-    <description>A controller-focused nighttime skin. Almost all controls can be hidden with skin menu.</description>
+    <version>2.3.0.01</version>
+    <description>A very flexible, controller-focused nighttime skin with stacked waveforms,
+4 decks and 64 samplers.</description>
```

Comment:
I think the manual line break is undesired here.

Reply:
Without, long strings make the window grow or enable scrollbars which looks not that good.
Until I figure out how to apply line breaks automatically I'd leave it this way.

Reply:
limiting the label width to that of the preview images should do the trick. I'll try

Reply:
auto line-break and mxWidth are in place now.